source/includes/admonitions/connection-resets.rst

```diff
@@ -0,0 +1,9 @@
+.. important::
+
+   Methods which reset the shell's connection to the server trigger a
+   termination of all open :manual:`sessions </reference/server-sessions>`.
```

Comment:
It may also be worth mentioning:

- Killing a session kills any in-progress operations in the session.
- Killing a session means the client will have to reauthenticate if auth is enabled.

Reply:
Done.